_data/FAQ.yml

```diff
@@ -0,0 +1,16 @@
+- question: Is HHEU affiliated with Major League Hacking (MLH)?
+  answer: >-
+    Yes and No! we are a completely seperate entiry but currently use MLH resources to allow our events to run.
+
+- question: Is it free?
+  answer: >-
+    HHEU tries to keep all events open to everyone by making our events free and offer reasonable travel reimbursement where possible
+    <small>Our events are funded from sponsors and uses resources from both [HackNotts](https://www.hacknotts.com/)' and
+    [DurHack](https://durhack.com)</small>
```

Comment:
This is great but definitely a little too early to publish to the world, me thinks 😅 How about this?

> HHEU wants to support the hackathon community in the UK and Europe. In the past, events like our [HHEU Unconference](/HHEUConf.html) have been free to attend and have offered travel reimbursements.
> 
> HHEU Unconference was kindly supported by sponsors and used resources from [HackNotts](https://www.hacknotts.com/) and [DurHack](https://durhack.com).